interpreter/custom/handler_custom.ml

```diff
@@ -159,7 +166,7 @@ and arrange_sec = function
 let check m custom =
   let {place; _} = custom.it in
   assert (compare_place place (After Custom) > 0);
-  match decode_content m custom with
+  match decode_content m "" custom with
```

Comment:
There is really no other option, and it's a consequence of the fact that `@custom` cannot roundtrip all sections:
If the original form of the module was text, there is no binary at all.
If a section would need the binary here , it means that it should not be used with `@custom`.

At the same time, there is no use for the `bs` parameter to `decode` as of now (and maybe ever? it would mean that a section needs some data from the binary that is not captured by the ast), so I could just remove it.
And at the very least, for consistency, `decode_content` should always pass an empty string to `Handler.decode` , I guess?

Reply:
I removed the second parameter of decode_content based on the above reasoning